### PR TITLE
[JBTM-3867] CI: Do not use WildFly latest with Java 11

### PR DESCRIPTION
**Pull request by marcosgopen**
https://issues.redhat.com/browse/JBTM-3867

CORE TOMCAT AS_TESTS RTS JACOCO XTS QA_JTA QA_JTS_OPENJDKORB PERFORMANCE LRA DB_TESTS mysql db2 postgres oracle

JDK17

### CODE REVIEW

**Comment by marcosgopen** [comment]
@mmusgrov I would like to propose to switch the default JDK (to 17) for new PRs and update the PULL_REQUEST_TEMPLATE.md, so that new PRs will run the AS tests too. WDYT?

File: scripts/hudson/narayana.sh [hunk 1]
```diff
@@ -397,8 +406,7 @@ function build_as {
   if [ ! -d  ${WILDFLY_CLONED_REPO}/dist/target/wildfly-${WILDFLY_VERSION_FROM_JBOSS_AS} ]; then
 
     # building WildFly
-    [ "$_jdk" -lt 17 ] && export MAVEN_OPTS="-XX:MaxMetaspaceSize=512m -XX:+UseConcMarkSweepGC $MAVEN_OPTS"
-    [ "$_jdk" -ge 17 ] && export MAVEN_OPTS="-XX:MaxMetaspaceSize=512m $MAVEN_OPTS"
+    export MAVEN_OPTS="-XX:MaxMetaspaceSize=512m $MAVEN_OPTS"
```
Comment (mmusgrov) [inline scripts/hudson/narayana.sh:409]: Well spotted

**Review by mmusgrov — APPROVED** [review]

**Comment by mmusgrov** [comment]
@marcosgopen The axis failed with `Could not find artifact org.eclipse.m2e:lifecycle-mapping:pom:1.0.0 in central`.

But we cannot make progress on PRs unless we merge this PR so I am going to merge it with this failure and we can work on it independently.

**Comment by marcosgopen** [comment]
> @marcosgopen The axis failed with `Could not find artifact org.eclipse.m2e:lifecycle-mapping:pom:1.0.0 in central`.
> 
> But we cannot make progress on PRs unless we merge this PR so I am going to merge it with this failure and we can work on it independently.

I agree, thanks @mmusgrov

**Pull request merged**